File: JS/URI/1035.js
const { Console } = require('console');

const input = require('fs').readFileSync('stdin', 'utf8');
const lines = input.split('\n');


const linha = lines[0];

const linhas = linha.split(' ');

    if((linhas[0]%2===0) && (linhas[1]>linhas[2]) && (linhas[2]+linhas[3] > linhas[0]+linhas[1]) && (linhas[3]>linhas[0]) && (linhas[2] > 0) && (linhas[3] > 0)){
        console.log("Valores aceitos");
    }
    else
        console.log("Valores nao aceitos");

File: Python/URI/1035.py
linhas = input().split(" ")

a, b, c, d = linhas

if (int(a)%2==0) and (int(b)>int(c)) and (int(c)+int(d) > int(a)+int(b)) and (int(d)>int(a)) and (int(c) > 0) and (int(d) > 0):
    print("Valores aceitos")
    
else:
    print("Valores nao aceitos")